README.md

```diff
@@ -135,23 +134,6 @@ yarn serve
   ```
   Why?  See [this issue](https://github.com/microsoft/TypeScript/issues/40878) for an explanation.
 
-- Advanced database commands:
-
-  ```bash
-  # Download & import USA data files.  Also create all other countries.
-  yarn refresh-db full #  Remove 'full' to import only a small dataset
-
-  # Re-import USA from previously downloaded data files (cache dir: ./tmp)
-  # Note: this command will drop and recreate the 'areas' and 'climbs' collection.
-  yarn seed-usa
-
-  # Add all countries (except for USA)
-  yarn add-countries
-
-  # Update area statistics (can be rerun as needed)
-  yarn update-stats
-  ```
-
```

Comment:
Are these being deprecated?

Reply:
I only removed them from the README. I'm voting for deprecating/remove them completely.